sentences/de/binary_sensor_HassGetState.yaml

```diff
@@ -371,3 +371,14 @@ intents:
           domain: binary_sensor
           device_class: heat
           state: "on"
+
+      # Safety
+      - sentences:
+          - "(ist|sind) <name> {bs_safety_states:state} [<area>]"
```

Comment:
we need also some tests for area aware sentences

Reply:
Sure, done 😃